examples/pt2/torch_export_aot_compile/README.md

```diff
@@ -0,0 +1,69 @@
+# TorchServe inference with torch._export aot_compile
+
+This example shows how to run TorchServe with Torch exported model with AOTInductor
+
+Using `torch.compile` to wrap your existing eager PyTorch model can result in out of the box speedups. However, `torch.compile` is a JIT compiler. TorchServe has been supporting `torch.compile` since PyTorch 2.0 release. In a production setting, when you have multiple instances of TorchServe, each of of your instances would `torch.compile` the model on inference. TorchServe's model archiever is not able to truly guarantee reproducibility because its a JIT compiler.
+
+In addition, the first inference request with `torch.compile` will be slow as the model needs to compile.
+
+To solve this problem, `torch.export` has an experimental API `torch._export.aot_compile` which is able to `torch.export` a torch compilable model with no graphbreaks along with AOTInductor.
+
+You can find more details [here](https://pytorch.org/docs/main/torch.compiler_aot_inductor.html)
+
+
+
+### Pre-requisites
+
+- `PyTorch >= 2.2.0`
+- `CUDA 12.1`
+
+Change directory to the examples directory
+Ex:  `cd` to `examples/pt2/torch_export_aot_compile`
+
+Install PyTorch 2.2 nightlies by running
+```
+chmod +x install_pytorch_nightlies.sh
+source install_pytorch_nightlies.sh
+```
+
+### Create a Torch exported model with AOTInductor
+
+The model is saved with `.so` extension
+Here we are torch exporting with AOT Inductor with `max_auotune` mode.
+This is also making use of `dynamic_shapes` to support batch size from 1 to 32.
```

Comment:
did this interact well with dynamic batching? What happens if you set the batch size to something really large instead?

Reply:
tried with batch_size 16. It works. I guess we launch it as an experimental feature and then based on issues/feedback, address them